src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/live_event.py

```diff
@@ -4,14 +4,21 @@
 # --------------------------------------------------------------------------------------------
 
 
-def create(client, resource_group_name, account_name, live_event_name, streaming_protocol, location):
+def create(client, resource_group_name, account_name, live_event_name, streaming_protocol, location,
+           auto_start=False, encoding_type=None, preset_name=None, tags=None, description=None):
     from azure.mgmt.media.models import LiveEventInputProtocol
     from azure.mgmt.media.models import LiveEventInput
     from azure.mgmt.media.models import LiveEvent
+    from azure.mgmt.media.models import LiveEventEncodingType
+    from azure.mgmt.media.models import LiveEventEncoding
 
     protocol = LiveEventInputProtocol(streaming_protocol)
     liveeventinput = LiveEventInput(streaming_protocol=protocol)
-    live_event = LiveEvent(input=liveeventinput, location=location)
+    encodingtypeenum = LiveEventEncodingType(encoding_type)
```

Comment:
This is not necessary and it's throwing an error. when you try to create a live event without an encoding type. You shuold pass the encoding_type value to the encoding_type constructor parameter of LiveEventEncoding model.